server/models/Conversation.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const conversationSchema = new Schema({
+  userOne: { type: Schema.Types.ObjectId, ref: "User" },
+  userTwo: { type: Schema.Types.ObjectId, ref: "User" },
```

Comment:
Even if we're assuming only two users can join a conversation for now, we might want to consider designing the schema such that we could have an arbitrary number of users per conversation in the future.

Reply:
Ah yeah good thinking, I have refactored to making users an array of refs to facilitate further expansion if desired.